assets/textures/gui/bevel/autoevo.png.import

```diff
@@ -0,0 +1,35 @@
+[remap]
```

Comment:
I think the icon file should be named something like "autoEvoPending.png" to be more clear what it is (it's not a general icon used to refer to auto-evo).